wgengine/wgcfg/writer.go

```diff
@@ -49,13 +49,29 @@ func (cfg *Config) ToUAPI(logf logger.Logf, w io.Writer, prev *Config) error {
 	// Add/configure all new peers.
 	for _, p := range cfg.Peers {
 		oldPeer, wasPresent := old[p.PublicKey]
+
+		// We only want to write the peer header/version if we're about
```

Comment:
It might be a good defense to always have these be true, unconditionally, if `!wasPresent`, as they're reflecting on a zero-value, and not setting the zero value may imply a default that isn't strictly zero.

Reply:
This is technically a behavioural change, but I'm pretty sure it doesn't matter in practice since the zero values would always compare as non-equal.